src/react/MessageFormatted.tsx

```diff
@@ -80,7 +86,7 @@ export const MessagePart = ({ part, ...props }: { part: MessageFormatPart } & Co
     obfuscated && messageFormatStylesMap.obfuscated
   ].filter(a => a !== false && a !== undefined).filter(Boolean)
 
-  return <span title={hoverItemText} style={parseInlineStyle(applyStyles.join(' '))} {...clickProps} {...props}>{text}</span>
```

Comment:
that makes sense for me, but wait, how did it work before then

Reply:
I don't thirk it did. Only one style should've been able to apply.